SECURITY.md

```diff
@@ -0,0 +1,78 @@
+# Helm Security Process and Policy
+
+This document provides the details on the Helm security policy and details the processes
+surrounding security handling including a how to guide on reporting a security vulnerability
+for anything within the Helm organization.
+
+## Report A Vulnerability
+
+We’re extremely grateful for security researchers and users that report vulnerabilities
+to the Helm community. All reports are thoroughly investigated by a set of Helm maintainers.
+
+To make a report please email the private security list at cncf-helm-security@lists.cncf.io
+with the details.
+
+You may, but are not required to, encrypt your email to this list using the PGP keys
+of security team members, listed below.
+
+Name               | Key URL                                              | Fingerprint
+------------------ | ---------------------------------------------------- | -----------
+Adam Reese         | [link](https://keybase.io/adamreese/pgp_keys.asc)    | 49D0 9C86 C3DC 8DA3 F0A0 7622 1EF6 1234 7F8A 9958
+Adnan Abdulhussein | [link](https://keybase.io/prydonius/pgp_keys.asc)    | 5111 DA73 DF12 D8E8 12CA 462F 2CDB BFBB 37AE 822A
+Justin Scott       ||
+Matt Butcher       | [link](https://keybase.io/technosophos/pgp_keys.asc) | ABA2 5295 98F6 626C 420D 335B 62F4 9E74 7D91 1B60
+Matt Farina        | [link](https://keybase.io/mattfarina/pgp_keys.asc)   | 672C 657B E06B 4B30 969C 4A57 4614 49C2 5E36 B98E
+Matt Fisher        | [link](https://keybase.io/bacongobbler/pgp_keys.asc) | 967F 8AC5 E221 6F9F 4FD2 70AD 92AA 783C BAAE 8E3B
+Michelle Noorali   ||
+Nikhil Manchanda   | [link](https://keybase.io/slicknik/pgp_keys.asc)     | 5674 7211 71F8 CD59 F285 353D 9692 757E BCD3 D66C
+Vic Iglesias       | [link](https://keybase.io/vicnastea/pgp_keys.asc)    | 43A7 994E F6C7 7D7F 147B C152 381B ABCF F47D EAFC
+
+### When To Send A Report
+
+You think you have found a vulnerability in a Helm project or a dependency of a Helm project. This can be any of the repositories on the [Helm GitHub organization](https://github.com/helm).
+
+### When Not To Send A Report
+
+* If a vulnerability has been found in an application deployed by a Helm Chart. Instead, contact the application maintainers
+* For guidance on securing Helm. Please see the documentation or ask in one of the support channels
+* You are looking for Helm applying security updates
+
+### Security Vulnerability Response
+
+Each report will be reviewed and receipt acknowledged within 3 business days. This will set off the security review process detailed below.
+
+Any vulnerability information shared with the security team stays within the Helm project and will not be shared with others unless it is necessary to fix the issue. Information is shared only on a need to know basis.
```

Comment:
Should we also add that we ask the person disclosing the vulnerability to not publicly disclose the vulnerability until it has been fixed? To me this reads about how we handle a security vulnerability, but doesn't explicitly mention how we'd like the person disclosing the report to act.

Reply:
Agree. Could we say: 

> We ask that security researchers act in good faith by not disclosing the issue to others. And we strive to act in good faith by acting swiftly, and by justly crediting the researcher(s) in writing.

Reply:
I swapped out "security researcher" for "vulnerability reporter". Otherwise it looks good. Thanks for catching that.